Makefile

```diff
@@ -61,6 +63,21 @@ install_system_db_dependencies: apt_get_update ## Install system-level DB depend
 
 install_system_dependencies: apt_get_update ## Install system-level dependencies from `debian_packages.lst`. Ignores comments.
 	sudo -E apt-get install -y `grep -v '^#' debian_packages.lst | tr -d '\r'`
+	if [ -z $$CI ] ; then \
+		echo "Installing Firefox because we're not in a CI."; \
+		sudo apt-get install -y libgtk3.0-cil-dev libasound2 libasound2 libdbus-glib-1-2 libdbus-1-3; \
```

Comment:
I've created a patch for the deployment playbook `ocim-vagrant` to install those but I think we might as well keep this here. I just recommend adding `no-install-recommends` to prevent installed unnecessary packages.